challenge_6.js

```diff
@@ -6,11 +6,35 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
   this.naturalNumbers = naturalNumbers;
+
+  this.sumOfSquares = function () {
+    let currentNum = 1;
+    let sum = 0;
+    while (currentNum <= this.naturalNumbers) {
+      let squareOfCurrentNum = Math.pow(currentNum, 2);
+      sum += squareOfCurrentNum;
```

Comment:
You could combine this line and the line above into one line since `squareOfCurrentNum` isn't used anywhere else in this while loop.